src/components/todo.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+<div id="todo">
+  <h1>ToDoリスト</h1>
+  <input type="radio" v-model="filter" name="group" value="all">すべて
+  <input type="radio" v-model="filter" name="group" value="working">作業中
+  <input type="radio" v-model="filter" name="group" value="finish">完了
+
+  <table>
+    <tr>
+      <th>ID</th>
+      <th>コメント</th>
+      <th>状態</th>
+    </tr>
+    <tbody v-for="(todo,index) in filteredTodos" :key="index">
+      <th>{{todo.i}}</th>
+      <th>{{todo.item}}</th>
+      <th><button @click="taskChange(todo.i)">{{todo.state}}</button></th>
+      <th><button @click="deleteItem(todo.i)">削除</button></th>
+    </tbody>
+  </table>
+
+  <h2>新規タスクの追加</h2>
+  <form @submit.prevent>
+    <input type="text" v-model="newItem">
+    <button @click="addItem">追加</button>
+  </form>
+</div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      newItem: '',
+      todos: [],
+      filter: 'all'
+    }
+  },
+  computed: {
+    filteredTodos() {
+      if(this.filter == 'all') {
```

Comment:
等価の条件の場合、[厳密等価演算子](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Operators/Strict_equality)を利用するとより良いです。
```suggestion
      if(this.filter === 'all') {
```
他の箇所も同様ですので、修正してみましょう。